src/markdown-convert.ts

```diff
@@ -202,6 +204,8 @@ export async function markdownConvert(
     usePandocParser: boolean;
     imageMagickPath: string;
     mermaidTheme: string;
+    onWillTransformMarkdown?: any;
```

Comment:
I would suggest we try to declare a specific type here.

Reply:
How to? Can we declare a function type?

Reply:
Done.

Reply:
Do you actually have tests for the mume engine which converts markdown and compares to the reference results??

Reply:
@gabyx No we don't 😂